### PR TITLE
Update minishift directory structure to automout script

### PR DESCRIPTION
**Pull request by praveenkumar**
This is just to add minishift related directory as part of automount script.

### CODE REVIEW

**Comment by praveenkumar** [comment]
@minikube-bot ok to test

File: deploy/iso/minikube-iso/package/automount/minikube-automount [hunk 1]
```diff
@@ -181,6 +181,10 @@ if [ -n "$BOOT2DOCKER_DATA" ]; then
     mkdir -p /mnt/$PARTNAME/var/lib/localkube
     mkdir /var/lib/localkube
     mount --bind /mnt/$PARTNAME/var/lib/localkube /var/lib/localkube
+
+    mkdir -p /mnt/$PARTNAME/var/lib/minishift
```
Comment (budhrg) [inline deploy/iso/minikube-iso/package/automount/minikube-automount]: :+1:

**Comment by r2d4** [comment]
I added you guys to the whitelist, so PRs should start testing automatically now

@minikube-bot ok to test

**Comment by praveenkumar** [comment]
@r2d4 Can we merge this now?

**Comment by r2d4** [comment]
@minikube-bot retest this please

**Comment by r2d4** [comment]
Yup it should pass the cross build now - I added a new integration slave, but I didn't have all the correct utilities needed to build the ISO on it.  Should be fixed now

**Pull request merged**